fuzzers/054-pip-fan-alt/Makefile

```diff
@@ -0,0 +1,6 @@
+MAKETODO_FLAGS=--re "^INT_[LR].FAN_ALT.*GFAN"
+GENERATE_FLAGS=--todo ../todo.txt
+N = 48
+SEGMATCH_FLAGS=-m 30 -M 45 -c 5
```

Comment:
Why -c 5?  -c is only set bits I believe?

Reply:
Yep, my mistake, it was a leftover for an ulterior check